Feature/720 check class exists on lookup

robbieaverill opened the pull request:
Add info message to `dev:class:lookup` output when the resolved class doesn't exist.

References issue #720 .

robbieaverill commented:
Looks like I took some commits from master with me. If you'd like them excluded, let me know and I'll rebase.

ktomk commented:
It looks like you've created your feature branch from master. Please checkout the feature branch and rebase it on top of develop. Pull develop first so that the rebase works best.

Also please squash the typo fix into the previous commit.

Afterwards the PR should have one commit.

Also have you considered to write a unit-test to check the new behaviour?

robbieaverill commented on readme.rst:
```diff
@@ -1276,7 +1278,7 @@ Rewrite List
 Lists all registered class rewrites.
 
-.. code-blocks:: sh
+.. code-block:: sh
```
Sorry guys, this one still came with me :)

ktomk commented:
@robbieaverill Just keep that readme.rst file out of your commit and it should be fine. This might be caused be a stale develop branch, I was fixing an earlier merge by moving it from the master branch into develop.

robbieaverill commented:
Hi @ktomk 

I have rebased onto develop and updated the unit tests for `ClassLookupCommand` by adding a data provider to pass in the type, dev code (name) and expected result, as well as whether the class should exist or not. If it shouldn't exist, we assert that the message contains a "does not exist" message as well.

The provider also covers helpers and blocks now.

Cheers,
Robbie

ktomk commented:
@robbieaverill: Thanks a lot for all your efforts! Tests look much better!

robbieaverill commented:
No worries, glad to contribute.

On Tue, Nov 3, 2015 at 11:21 AM, Tom Klingenberg <notifications@github.com>
wrote:

> @robbieaverill <https://github.com/robbieaverill>: Thanks a lot for all
> your efforts! Tests look much better!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/netz98/n98-magerun/pull/721#issuecomment-153176383>.
>



-- 
*Robbie Averill*
LA PETITE MANOUCHE - NZ Gypsy Jazz Guitar Duo
Phone: +64 27 321 4758
contact@lapetitemanouche.com
www.lapetitemanouche.com

ktomk commented:
no worries at all ... https://www.youtube.com/watch?v=es6EOIHIsBI ... ;)

robbieaverill commented:
Haha - my second life
